### DISCUSSION TITLE
[Kavita] Integrate Kurp (upscalling comic) into Kavita docker

### DISCUSSION THREAD

**Discussion by Gunnm92** [Kareadita/Kavita#2825] [Ideas]
### Idea Description

integrate Kurp (https://github.com/Snd-R/kurp) into Kavita docker to provide ability to upscale comic by using the proxy port

### Idea Category

API

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
This is not something I have interest in doing.
